sklearn/preprocessing/_encoders.py

```diff
@@ -551,6 +553,8 @@ class OrdinalEncoder(_BaseEncoder):
 
     Read more in the :ref:`User Guide <preprocessing_categorical_features>`.
 
+    .. versionchanged:: 0.20.1
```

Comment:
How did you decide between versionadded and versionchanged here? I think adding versionadded everywhere would be good, and we could do additional versionchanged ones. But then the versionchanged should say what changed, otherwise it's not very helpful.

Reply:
@amueller My criteria was that it seemed there was a significant bug #12365 that was fixed in version 0.20.1